srcpkgs/xdm/files/xdm.pam

```diff
@@ -6,3 +6,4 @@ password	required	pam_unix.so
 session		required	pam_unix.so
 session		required	pam_limits.so
 session		required	pam_loginuid.so
+session		optional	pam_elogind.so
```

Comment:
I believe this should use `-session` instead. It's what `system-login` uses.

Reply:
Good idea on the “-“. Apologies that this PR is a mess because of my stubborn experiment in doing it online without actually using git, which is a fail.  I will clean it up in a day or two, or just close it and reissue a clean version.

Reply:
No problem. I can fix and merge it myself, if you want.

Reply:
Go ahead ericonr,  I was going to have to work out how to merge or rebase and squash this when I did not create it from a downloaded repository to begin with.  Thanks!